db/migrate/20190205142619_add_content_constraints.rb

```diff
@@ -0,0 +1,37 @@
+class AddContentConstraints < ActiveRecord::Migration[5.2]
+  def up
+    Katello::ProductContent.where(:content_id => nil).delete_all
+    Katello::ProductContent.where(:product_id => nil).delete_all
+
+    change_column :katello_product_contents, :content_id, :integer, :null => false
+    change_column :katello_product_contents, :product_id, :integer, :null => false
+
+    duplicates =  ActiveRecord::Base.connection.execute(
```

Comment:
Think you can just say execute ? -> https://github.com/Katello/katello/blob/master/db/migrate/20171114150937_cleanup_installed_packages.rb#L14

Reply:
Might want to add reset_column_information after change columns